source/trendfiguur/03_inschatting.Rmd

```diff
@@ -0,0 +1,605 @@
+  id = "f(
+      id, model = 'iid',
+      hyper = list(theta = list(prior = 'pc.prec', param = c(0.5, 0.01)))
+    )"
+) |>
+  mutate(model_id = row_number()) |>
+  pivot_longer(-"model_id", names_to = "variable", values_to = "selected") |>
+  filter(.data$selected != "") |>
+  group_by(.data$model_id) |>
+  summarise(
+    formula = paste(.data$selected, collapse = " + ") |>
+      sprintf(fmt = "correct ~ %s") |>
+      map(as.formula)
+  ) |>
+  mutate(
+    model = map(
+      .data$formula,
+      ~inla(
+        .x, family = "binomial", data = ds_interpretatie,
+        control.compute = list(waic = TRUE), control.predictor = list(link = 1)
+      )
+    ),
+    waic = map(.data$model, "waic") |>
+      map_dbl("waic"),
+    waic = .data$waic - min(.data$waic),
+    weight = exp(-.data$waic / 2),
+    weight = .data$weight / sum(.data$weight)
+  ) -> model_fit
+```
+
+```{r verklaring-voorspelling}
+model_fit |>
+  transmute(
+    .data$weight,
+    predictions = map(.data$model, "summary.linear.predictor") |>
+      map(
+        ~head(.x, n_pred) |>
+        select("mean", "sd") |>
+        rownames_to_column()
+    )
+  ) |>
+  unnest("predictions") |>
+  mutate(
+    estimate = .data$weight * .data$mean,
+    variance = (.data$weight * .data$sd) ^ 2
+  ) |>
+  group_by(.data$rowname) |>
+  summarise(estimate = sum(.data$estimate), sd = sqrt(sum(.data$variance))) |>
+  mutate(
+    median = plogis(.data$estimate),
+    lcl = plogis(qnorm(0.025, .data$estimate, .data$sd)),
+    ucl = plogis(qnorm(0.975, .data$estimate, .data$sd))
+  ) |>
+  bind_cols(
+    ds_interpretatie |>
+      filter(is.na(.data$correct))
+  ) -> voorspelling
+```
+
+Tijdens de eerste reeks gaven de deelnemers in gemiddeld vier van de tien keren het juiste antwoord (figuur \@ref(fig:verklaring-beste)).
+Tijdens de tweede reeks steeg dit tot drie kwart van de gevallen.
+Vermoedelijk omdat het merendeel van de deelnemers tijdens de tweede reeks betrouwbaarheidsintervallen en referentielijnen verkoos.
+In mindere mate stellen we nog een effect vast van de schaal van de y-as: deelnemers die voor een procentuele wijziging kozen geven vaker een correct antwoord.
+Ook het tonen van de symbolen verhoogt de kans op een correct antwoord.
+
+Strikt genomen onderzochten we met welke stijl van figuren de deelnemers het best instaat zijn om tot een gelijkwaardige interpretatie als de makers van de figuur te komen.
+Het correcte antwoord selecteren we bijgevolg louter op basis van de volledige set van gegevens, inclusief de kennis over onzekerheid en referentielijnen.
+Indien we de stijl van de figuur in rekening brengen, zou de correctie interpretatie van de eerste reeks figuren steeds "geen idee" zijn.
+
+```{r verklaring-beste, fig.cap = "Gemiddelde kans op een correcte interpretatie in functie van het gebruik van symbolen en de schaal van y-schaal. Het tweede reeks valt grotendeels samen met het toevoegen van referentielijnen en weergave van onzekerheid."}
```

Comment:
```suggestion
```{r verklaring-beste, fig.cap = "Gemiddelde kans op een correcte interpretatie in functie van het gebruik van symbolen en de schaal van y-as. De tweede reeks valt grotendeels samen met het toevoegen van referentielijnen en weergave van onzekerheid."}
```